packages/core/contracts/test/ignore-errors/Constructors.sol

```diff
@@ -4,7 +4,8 @@ pragma solidity ^0.8.9;
 abstract contract UnsafeParent {
     /// @custom:oz-upgrades-unsafe-allow constructor
     constructor(bytes memory data) {
-        msg.sender.delegatecall(data);
+       (bool result,)= msg.sender.delegatecall(data);
+        require(result == true, "delegatecall failed");
```

Comment:
Please apply this suggestion in the other contracts as well.

```suggestion
       (bool result,) = msg.sender.delegatecall(data);
        require(result, "delegatecall failed");
```

Reply:
Sure. 

- Some other files in `packages/core/` were resolving this by simply writing `s;` of returned boolean value `s`. I have added similar logic in those files too for code consistency which also gives a helpful message `"delegatecall failed"` when the function call fails.

The commit changes, [here.](https://github.com/OpenZeppelin/openzeppelin-upgrades/commit/7ffa8d86cd6416019189decb73b1a9ad59d0f350
)

PS: 
<img width="959" alt="Screenshot 2023-01-29 at 10 47 45 AM" src="https://user-images.githubusercontent.com/73742938/215306879-5cfeaa69-fae5-4838-b5b2-a74c5b8c5833.png">